.github/workflows/codechecker-analysis.yml

```diff
@@ -86,7 +88,7 @@ jobs:
             --slave                                            \
               /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-$LLVM_VER
           update-alternatives --query clang
-          echo "::set-output name=LLVM_REAL_VERSION::$(clang --version | head -n 1 | cut -d' ' -f3-)"
+          echo "::set-output name=LLVM_REAL_VERSION::$(clang --version | head -n 1 | cut -d' ' -f4-)"
```

Comment:
Turns out that the nightly PPA binaries output their format in a wholly different format than one built locally. :joy: This removes the word `version` from `[...]-llvm-`~`version-`~`13.0.0-[...]` in the artefact name.